app/dao/tokens_dao.py

```diff
@@ -0,0 +1,22 @@
+from app import db
+from app.models import Token
+
+
+def save_token_model(token, update_dict={}):
+    if update_dict:
+        del update_dict['id']
+        db.session.query(Token).filter_by(id=token.id).update(update_dict)
+    else:
+        db.session.add(token)
+    db.session.commit()
+
+
+def get_model_tokens(service_id=None):
```

Comment:
When is the need for more than one token? Should a service have more than one token?